Forward-port of the following upstream patch:
From cfc0e1a7a7fe69e30c73c49afccc4a1dd4a139e0 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Thu, 27 Aug 2020 09:53:54 -0700
Subject: [PATCH 2/2] UBUNTU: SAUCE: enable pinning of pre AppArmor 3.x policy

Ubuntu several packages that are shipping policy that has not been
updated for AppArmor 3 and abi rules.

Pin the policy to the out of tree networking abi as Ubuntu carried
the out of tree networking patches. Pinning the policy is needed
to address two issues.
1. Silence policy load warnings

   Warning from stdin (stdin line 1): apparmor_parser: File 'example'
   missing feature abi, falling back to default policy feature abi.

2. Ubuntu policy expects to be able to use network, unix, and dbus
   rules but without pinning the abi these rules will either be
   silently dropped or the policy will fail to compile.

Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 parser/parser.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/parser/parser.conf
+++ b/parser/parser.conf
@@ -76,4 +76,4 @@
 #policy-features=/etc/apparmor.d/abi/kernel-5.4-vanilla
 
 ## Pin older policy to the 5.4 kernel abi + out of tree network and af_unix
-#policy-features=/etc/apparmor.d/abi/kernel-5.4-outoftree-network
+policy-features=/etc/apparmor.d/abi/kernel-5.4-outoftree-network
